ethstorage/storage_manager.go

```diff
@@ -70,8 +70,8 @@ type StorageManager struct {
 	DownloadThreadNum int
 	shardManager      *ShardManager
 	localL1           int64      // local view of most-recent-finalized L1 block
-	mu                sync.Mutex // protect lastKvIdx, shardManager and blobMeta read/write state
-	lastKvIdx         uint64     // lastKvIndex in the most-recent-finalized L1 block
+	mu                sync.Mutex // protect kvEntryCount, shardManager and blobMeta read/write state
+	kvEntryCount      uint64     // kvEntryCnt in the most-recent-finalized L1 block
```

Comment:
It’s confusing to use both kvEntryCount and kvEntryCnt. I’d suggest standardizing on a single name to avoid potential bugs.